pages/index.js

```diff
@@ -86,34 +72,7 @@ export default function Home({ posts, status }) {
                 >
                   <div className="rounded-t-lg overflow-hidden relative pt-[52%]">
                     <div className="absolute inset-0">
-                      <div className="relative">
-                        {post.image.data.map((image) => (
-                          <>
-                            <Link
-                              key={image.attributes.id}
-                              href={"/post/" + post.slug}
-                            >
-                              <Image
-                                layout="responsive"
-                                objectFit="contain"
-                                width={500}
-                                height={500}
-                                src={image.attributes.url || ""}
-                                alt={image.alternativeText || ""}
-                              />
-                            </Link>
-                          </>
-                        ))}
-                        <Link
-                          href={
-                            "/categories/" +
-                            post.categories.data.attributes.slug
-                          }
-                          className="px-5 py-2 bg-pink-600 rounded-lg absolute top-[15px] right-[20px] cursor-pointer z-10 text-white hover:text-white active:scale-[0.98]"
-                        >
-                          {post.categories.data.attributes.name}
-                        </Link>
-                      </div>
+                      <div className="relative"></div>
```

Comment:
Let's comment out the code, instead of removing it. As it will be used for getting all blogs